LICENSE.rst

```diff
@@ -0,0 +1,41 @@
+This program is part of the DiffPy and DANSE open-source projects
```

Comment:
This one should be the BSD-3clause.  We own copyright so we don't want the old license.

Reply:
I copied over the old license from this package, but we want to replace it with the cookiecutter license?

Reply:
Normally no, but in this case yes.... 👍